src/client.js

```diff
@@ -33,18 +34,21 @@ const Client = function Client(config = {}) {
   };
 
   const batch = FirehoseBatcher.getInstance(clientConfig.get(), (params, batcher) => {
-    return restAPI(batcher.config, "firehose", "post", params);
+    return restAPI(batcher.config, "firehose", "post", params, {
+      timeout: process.env.BATCH_TIMEOUT || 15000,
```

Comment:
Let's use a smaller timeout here since we're retrying and building idempotent messages...